dis_006_reliefweb_disasters/contents/src/__init__.py

```diff
@@ -55,263 +72,445 @@
                         elif key=='primary_country':
+                            # get the iso3 country code for primary country from 'iso3' feature
                             item = entry['fields']['primary_country']['iso3']
+                        # if we are fetching data for featured column 
+                        elif key=='featured':
+                            # get the data for this column from 'featrued' feature which is a 
+                            # binary 'True' or 'False' value and covert it to a string
+                            item = str(entry['fields']['featured'])
+                        # if we are fetching data for country_name column
                         elif key=='country_name':
+                            # get the name of country from 'name' feature
                             item = country['name']
+                        # if we are fetching data for country_shortname column
                         elif key=='country_shortname':
+                            # get the short name for country from 'shortname' feature
                             item = country['shortname']
+                        # if we are fetching data for country_iso3 column
                         elif key=='country_iso3':
+                            # get the iso3 country code for secondary countries from 'iso3' feature
                             item = country['iso3']
+                        # if we are fetching data for current column
                         elif key== 'current':
+                            # get the data for this column from 'current' feature which is a 
+                            # binary 'True' or 'False' value and covert it to a string                            
                             item = str(entry['fields']['current'])
+                        # if we are fetching data for event_type_ids column
                         elif key == 'event_type_ids':
+                            # get secondary event ids from ids list
                             item = ids
+                        # if we are fetching data for event types column
                         elif key == 'event_types':
+                            # get secondary event names from names list
                             item = names
-                        elif key == 'url':
-                            item = entry['fields']['url']
-                        elif key == 'lon':
-                            item = country['location']['lon']
-                        elif key == 'lat':
-                            item = country['location']['lat']
+                        # if we are fetching data for event name column
+                        elif key=='event_name':
+                            # get the event name from 'name' feature
+                            item = entry['fields']['name']
+                        # if we are fetching data for coordinates
+                        elif key == 'lon' or 'lat':
```

Comment:
Make sure you test code changes to make sure they work as expected! If you had key='event', the statement you used ( key == 'lon' or 'lat') would have produced (False or 'lat') which would produce ('lat') which is not None, so the if statement would have gone through for any key, not just 'lon' or 'lat'.
```suggestion
                        elif key == 'lon' or key == 'lat':
```